config/locales/en/loans.en.yml

```diff
@@ -69,6 +69,13 @@ en:
       frozen: "Frozen"
       liquidated: "Liquidated"
     end_date: "End Date"
+    tab:
+      details: "Details"
+      questions: "Questions"
+      timeline: "Timeline"
+      timeline_list: "Timeline - List"
+      logs: "Logs"
+      calendar: "Calendar"
```

Comment:
Thanks for the tip @hooverlunch I did forget to clean out some of the old stuff. I usually comment stuff out, then go back and delete it. Obviously, there are issues with that method when whole files need to be removed. ;)